client/js/app/stores/ExplorerStore.js

```diff
@@ -128,7 +128,14 @@ function _getDefaultFilterCoercionType(explorer, filter) {
 function _prepareUpdates(explorer, updates) {
   // TODO: We're assigning the response object directly onto the model so we
   // don't have to loop through the (sometimes) massive response object.
-  var newModel = _.merge({}, explorer, _.omit(updates, 'response'));
+  function customizer(objValue, srcValue, key, object, source) {
```

Comment:
`object` and `source` aren't used anywhere in the function. It might be a bit cleaner to remove them from the argument list.